pkg/splunk/enterprise/indexercluster.go

```diff
@@ -445,8 +445,8 @@ func (mgr *indexerClusterPodManager) getClient(n int32) *splclient.SplunkClient
 	// Get Pod Name
 	memberName := GetSplunkStatefulsetPodName(SplunkIndexer, mgr.cr.GetName(), n)
 
-	// Get Fully Qualified Domain Name
-	fqdnName := splcommon.GetServiceFQDN(mgr.cr.GetNamespace(),
+	// Get uri
```

Comment:
I was wondering if the change from FQDN to URI here is required for disabling the default SSL, or if this change is an additional enhancement?